JapaneseFilter/sections/specific.txt

```diff
@@ -4,6 +4,8 @@
 ! /banner/banner_ - if it is necessary, make this rule domain specific
 !
 gyao.yahoo.co.jp##.ad-yads
+sankeibiz.jp##.module_recommend:has(> div.module_content > div.OUTBRAIN )
```

Comment:
wrong file and incorrect syntax

Reply:
This rule wasn't removed. Also it has unnecessary space after `div.OUTBRAIN` - it's incorrect syntax.

Reply:
still not fixed